### PR TITLE
Add a note about opt-in data in the Longitudinal dataset.

### PR DESCRIPTION
**Pull request by Dexterp37**
This comes from [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1405239).

### CODE REVIEW

File: datasets/batch_view/longitudinal/intro.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ In summary, the longitudinal table differs from `main_summary` in two important
 * The longitudinal dataset groups all data so that one row represents a client_id
 * The longitudinal dataset samples to 1% of all client_ids
 
+Please note that this dataset only contains pre-release (or opt-in) histograms and scalars.
```
Comment (harterrt) [inline datasets/batch_view/longitudinal/intro.md]: Isn't the opposite true? We [only include opt-out histograms](https://github.com/mozilla/telemetry-batch-view/blob/master/src/main/scala/com/mozilla/telemetry/views/Longitudinal.scala#L111) in longitudinal?

Comment (Dexterp37) [reply datasets/batch_view/longitudinal/intro.md]: Of course you're right :)

**Comment by mreid-moz** [comment]
Shouldn't this say "release channel (or opt-out)"?

**Comment by Dexterp37** [comment]
Right. Fixed with #64

**Pull request merged**